### PR TITLE
lizards have a little natural stabilization

### LINKED ISSUE #48876: Lizards are oneshot by temperature guns/watcher eye beams
Issue reported from  Round ID: 128456 (/tg/Station Manuel [ENGLISH] [US-WEST] [ROLEPLAY] [99% LAG FREE])

Reporting client version: 512.1484

Because of the body temperature changes in this PR https://github.com/tgstation/tgstation/pull/48569, lizards are apparently incapable of stabiling their temperature quickly enough not to die to the freeze ray shooting them just once. This weapon has an obscene number of shots, so you don't even have to try to hit someone with it.

I have no idea if this intentional or not because of the broken explanation in the PR body doesn't really help elucidate whether this is working as intended but it definitely is a massive, massive issue with ashlizards and lizard players in general.

I think the core problem is that they're not stabilizing quickly enough and that the temperature gun puts you at -170 degrees celcius. Which is....really fucking low. I think temp guns and watcher beams need a serious re-evaluation and pretty soon.

### PR DESCRIPTION
**Pull request by nightred**
## About The Pull Request

Gives lizards some natural body temperature stabilization.
If the lizard is at or over the hot/cold damage limit they will naturally stabilize to the safe zone.

## Why It's Good For The Game

Lizards being cold blooded is fun but it was to harsh,
They can now recover from very hot or cold like a normal human. but will not stabilize within the safe range. This adds some flair and challenge to being a lizard without dying as fast to cold things.

Closes #48876

## Changelog
:cl:
balance: lizards have some natural body temperature stabilization
/:cl:

### CODE REVIEW

**Comment by trollbreeder** [comment]
Man, i love evolution.

**Comment by improvedname** [comment]
There is this one kit called stimpack kit. that allows you know stabilize your temp piss fast but it never gets used because there was never a reason to.

**Comment by improvedname** [comment]
Like the real counter to this is litteraly super common and available in every coffee machine it even comes in the miner stimpacks

**Comment by kriskog** [comment]
The current state of this causes lizards to die if they gotta eva suit while cold. Good luck running to the coffee machine while cold slowed by the way.

**Comment by improvedname** [comment]
why are you gonna put on a eva suit when your having cold that only insulates the coldness even more taking it longer to get rid of

**Comment by kriskog** [comment]
We do use space suits as an emergency measure too, not just when we wanna fuck off to space to explore.

**Comment by nightred** [comment]
space suits do nothing to stabilize body temp they only hold it at exactly what it currently is.

**Comment by RaveRadbury** [comment]
What would it look like to balance the temperatures involved instead of fixing it this way?

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm [hunk 1]
```diff
@@ -29,9 +29,21 @@
 	bodytemp_heat_damage_limit = (BODYTEMP_HEAT_DAMAGE_LIMIT + 20) // This puts lizards 10 above lavaland max heat for ash lizards.
 	bodytemp_cold_damage_limit = (BODYTEMP_COLD_DAMAGE_LIMIT - 10)
 
-/// Lizards are cold blooded and do not stabilize body temperature naturally
+/// Lizards are cold blooded and do not stabilize body temperature naturally unless very cold
```
Comment (HomocidlePancakes) [inline code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: Comment says it only works when very cold, but it seems you stabilize it when very hot too

**Comment by zxaber** [comment]
We could put hot coffee into EVA. Or perhaps more thematically appropriate, some sort of warming pad that you like attach to your jumpsuit and it boosts your heat levels.

**Comment by ATH1909** [comment]
Yeah, hot coffee and the like _exists_, but how the fuck is a new player supposed to know that if they get cold as a lizardperson, they're supposed to run to a vending machine and dispense a hot drink? And do you really want to add "buy a hot drink from a vending machine" to the list of things lizardperson players have to do at the start of every round? That drink's gonna take up space in their inventory, too, and lizardpeople aren't really getting anything to compensate for the really hard nerf of being cold blooded. Being "more resistant to heat damage" is a joke of an upside and you know it; especially since lizards have a harder time cooling off once they're hot than humans do (I wouldn't be surprised if that makes lizards generally MORE vulnerable to heat damage than humans are, simply because humans can stabilize back to cooler temperatures faster).

This PR is an okay solution, and I really don't want us to take our chances with heating pads or whatever (because ash walkers will _totally_ have access to a heating pad and hot coffee). Merge this and be done with it.

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm (hunk 1 above)
Comment (nightred) [reply code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: Updated the comment

**Comment by optimumtact** [comment]
I don't agree with this PR and i'm okay with lizards being a fundamentally different choice.

**Comment by PKPenguin321** [comment]
then close it?

**Comment by optimumtact** [comment]
i'm not a dictator 😏

**Comment by HomocidlePancakes** [comment]
Can’t be a dictator when it comes to abusing lizards.

**Comment by ninjanomnom** [comment]
I'm with oranges here, some more sources for temperature regulation is probably merited though, like a temperature setting in EVA suits or something

**Comment by kriskog** [comment]
Since space is extremely cold, being exposed to space for even a few seconds will kill you as a lizard, despite getting in an eva suit just after. Heck, even running past a breach might be enough to crit you.

**Comment by ninjanomnom** [comment]
I'm not saying that this shouldn't be tweaked but rather that another solution is preferred

**Comment by XDTM** [comment]
Honestly temperature stabilization as a whole should be slower and based on ambient temperature, for other species as well. You can't really do temperature damage as it right now unless you either set the victim on fire or constantly push them to absolute 0 with space or frost oil, making less extreme mechanics like fever completely ineffective.

**Comment by zxaber** [comment]
>Since space is extremely cold, being exposed to space for even a few seconds will kill you as a lizard, despite getting in an eva suit just after. Heck, even running past a breach might be enough to crit you.

Is it time we add realism and remove space being magically cold?

**Comment by nightred** [comment]
I like the idea of increasing the stabilize divisor. Make it slower on all species.

**Comment by kriskog** [comment]
Part of the issue with space cold is that it makes you unnaturally cold. As if you'd be walking around with a core body temp of -150C (four ticks in space) even as a  non lizard.

**Pull request closed without merging**